engine/apps/heartbeat/models.py

```diff
@@ -43,9 +42,14 @@ class IntegrationHeartBeat(models.Model):
 
     created_at = models.DateTimeField(auto_now_add=True)
     timeout_seconds = models.IntegerField(default=0)
+
+    # Stores the latest received heartbeat signal time
     last_heartbeat_time = models.DateTimeField(default=None, null=True)
+    # Deprecated. This field is not used. TODO: remove it
     last_checkup_task_time = models.DateTimeField(default=None, null=True)
+    # Deprecated. Stored the latest scheduled `integration_heartbeat_checkup` task id. TODO: remove it
     actual_check_up_task_id = models.CharField(max_length=100)
```

Comment:
would it be safe to simply drop these two fields in this PR? Or at a minimum remove them from the model without dropping them from the db, so that a subsequent PR we can drop them? (Vadim [docs](https://github.com/grafana/oncall/pull/2659/files#diff-e14025c0fa40d4857e4b40fc96ea5ee995afe300014626b68cb55a479fa5b8fcR437-R453) have more info on this)